CDP4Authentication/Contracts/AuthenticatorPlugin.cs

```diff
@@ -84,6 +85,63 @@ protected virtual void LoadConfig()
             this.AuthenticatorConfig = this.DeserializeConfigFile(json, configLocation);
         }
 
+        /// <summary>
+        /// Write server salt value to config.json
```

Comment:
why do we need the server to write the salt to the config.json? that seems strange to me. This was intended to be a read only config file. I am afraid we are opening an attack vector here. Please organize a discussion on this.